source/_patterns/20-molecules/swipe-module/swipe-module.scss

```diff
@@ -0,0 +1,294 @@
+/*** LAYOUT ***/
+$outer-padding-small: 8px;
+$outer-padding: 32px;
+$inner-outer-padding: 16px;
+
+/*** SPACING ***/
+$gutter-padding-small: 8px;
+$gutter-padding: 12px;
+$half-gutter-padding-small: $gutter-padding-small / 2;
+$half-gutter-padding: $gutter-padding / 2;
+
+/*** UI ***/
+$swipe-remains-small: 48px;
+$swipe-remains: 52px;
+$swipe-remains-large: 92px;
+$row-count-tablet: 2;
+$row-count-desktop: 3;
+$item-max-width: 400px;
+
+.swipemod {
+  background: $color-bg-box;
+  padding: 24px $outer-padding-small 32px;
+  overflow-x: hidden;
+  
+  @include tablet-up {
+    padding: 24px $outer-padding 40px;
+  }
+}
+
+.swipemod-container {
+  max-width: $content-width;
+  margin: 0 auto;
+  padding: 20px 16px 4px;
+  background: $color-bg-caption;
+  border-radius: 2px;
+  box-shadow: 0 1px 2px 0 rgba(0,0,0,.24), 0 0 2px 0 rgba(0,0,0,.12);
+  position: relative;
+  
+  /*
+   * VISUAL ASSISTANCE FOR DEBUGING
+   *
+  @include tablet {
+    background-color: gray;
+    background-image: linear-gradient(90deg, transparent 50%, rgba(255,255,255,.5) 50%);
+    background-size: 100% 100%;
+  }
+  @include desktop-up {
+    background-color: gray;
+    background-image: linear-gradient(90deg, transparent 50%, rgba(255,255,255,.5) 50%);
+    background-size: 66.67% 66.67%;
+  }
+  */
+}
+
+.swipemod-titlecontainer {
+  display: flex;
+  justify-content: space-between;
+}
+.swipemod-title {
+  flex: 0 1 auto;
+  margin: 0;
+  font-family: $font-regular;
+  font-size: 22px;
+  line-height: 1.27;
+  letter-spacing: .2px;
+  color: $color-text-accentuation;
+}
+.swipemod-showall {
+  flex: 0 1 auto;
+  font-size: 16px;
+  font-family: $font-regular;
+  line-height: 1;
+  letter-spacing: .5px;
+  border-bottom: none;
+  top: 2px;
+  padding: 0 20px 0 0;
+  position: relative;
+
+  &:hover,
+  &:active,
+  &:focus {
+    border: none; // overwrite of global a-stlyes :rolling-eyes:
+  }
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 2px;
+    left: auto;
+    right: 6px;
+    height: 11px;
+    width: 11px;
+    border-bottom: 1px solid $color-link-default;
+    border-right: 1px solid $color-link-default;
+    transform: rotate(-45deg)
+  }
+  &:hover::before,
+  &:focus::before {
+    border-bottom-color: $color-link-hover;
+    border-right-color: $color-link-hover;
+  }
+  &:active::before {
+    border-bottom-color: $color-link-active;
+    border-right-color: $color-link-active;
+  }
+}
+
+.swipemod-swipecontainer {
+  display: flex;
+  overflow-x: auto;
+  margin: 0 -#{$outer-padding-small + $inner-outer-padding + $half-gutter-padding-small};
+  padding: 0 #{$outer-padding-small + $inner-outer-padding};
+
+  scroll-snap-type: mandatory;
+  scroll-snap-destination: 50% 0;
+  
+  @include tablet {
```

Comment:
Ist so wohl besser als ein tablet-up, da expliziter?

Reply:
Man könnte tablet und desktop auch mit tablet-up zusammennehmen, aber dann überschreiben wir das mit desktop-wide wieder … wenn nicht, ists zwar doppelt drin, dafür wird im Browser nichts dreifach überschrieben … ich hab mich in diesem Fall für jedes einzeln entschieden, da es dann auch sofort sichtbar ist wie es sich in jedem BP verhält. Oder was meint ihr?

Reply:
Genau das meinte ich mit meinem Kommentar, ich finde es so auch besser. Klar doppelt es sich leicht, aber in dem Fall bin ich ebenfalls für eine explizite Lösung.